packages/flutter/lib/src/widgets/form.dart

```diff
@@ -460,12 +468,18 @@ class FormField<T> extends StatefulWidget {
     this.onSaved,
     this.forceErrorText,
     this.validator,
+    this.errorBuilder,
     this.initialValue,
     this.enabled = true,
     AutovalidateMode? autovalidateMode,
     this.restorationId,
   }) : autovalidateMode = autovalidateMode ?? AutovalidateMode.disabled;
 
+  /// Function that returns the widget representing this form field. It is
+  /// passed the form field state as input, containing the current value and
+  /// validation state of this field.
```

Comment:
Nit: Would you mind breaking this apart so that the first sentence is its own paragraph? (Nit because I know you didn't write it in this PR.)